cluster-operator/src/main/java/io/strimzi/operator/cluster/model/AbstractModel.java

```diff
@@ -518,57 +556,59 @@ protected void setConfiguration(AbstractConfiguration configuration) {
      * @return The image name.
      */
     public String getImage() {
-        return this.image;
+        return image;
     }
 
     /**
-     * @return the service account used by the deployed cluster for Kubernetes/OpenShift API operations
+     * @return the name of the service account used by the deployed cluster for Kubernetes/OpenShift API operations.
      */
     protected String getServiceAccountName() {
         return null;
     }
 
     /**
-     * @return the cluster name
+     * @return the cluster name.
      */
     public String getCluster() {
         return cluster;
     }
 
     /**
      * Gets the name of a given pod in a StatefulSet.
-     * @param podId The Id of the pod.
+     *
+     * @param podId The Id (ordinal) of the pod.
      * @return The name of the pod with the given name.
      */
     public String getPodName(int podId) {
         return name + "-" + podId;
     }
 
     /**
-     * Sets the affinity as configured by the user in the cluster CR
+     * Sets the affinity as configured by the user in the cluster CR.
```

Comment:
Should we put empty line between the sentence and the @param? TBH I do not really mind, but it seems you did that in other Javadoc comments around.